src/w3d/renderer/dx8renderer.cpp

```diff
@@ -51,8 +286,1366 @@ void DX8MeshRendererClass::Clear_Pending_Delete_Lists()
+    unsigned int pass)
+{
+    int polycount = split_table.Get_Polygon_Count();
+    unsigned int index_count = 0;
+    int polygons = 0;
+
+    for (int i = 0; i < polycount; i++) {
+        if (split_table.Peek_Texture(i, pass, 0) == m_textures[0]) {
+            if (split_table.Peek_Texture(i, pass, 1) == m_textures[1]) {
+                if (Compare_Materials(split_table.Peek_Material(i, pass), m_material)) {
+                    if (split_table.Peek_Shader(i, pass) == m_shader) {
+                        polygons++;
+                    }
+                }
+            }
+        }
+    }
+
+    if (polygons) {
+        index_count = 3 * polygons;
+        TriIndex *polys = split_table.Get_Polygon_Array(pass);
+        DX8PolygonRendererClass *renderer = new DX8PolygonRendererClass(
+            index_count, split_table.Get_Mesh_Model_Class(), this, vertex_offset, index_offset, 0, pass);
+        m_polygonRendererList.Add_Tail(renderer);
+        IndexBufferClass::AppendLockClass lock(index_buffer, index_offset, index_count);
+        unsigned short *indices = lock.Get_Index_Array();
+        unsigned short vmax = 0;
+        unsigned short vmin = 0xFFFF;
+
+        for (int i = 0; i < polycount; i++) {
+            if (split_table.Peek_Texture(i, pass, 0) == m_textures[0]) {
+                if (split_table.Peek_Texture(i, pass, 1) == m_textures[1]) {
+                    if (Compare_Materials(split_table.Peek_Material(i, pass), m_material)) {
+                        if (split_table.Peek_Shader(i, pass) == m_shader) {
+                            unsigned short index = vertex_offset + polys[i].I;
+
+                            if (vmin >= index) {
+                                vmin = index;
+                            }
+
+                            if (vmax <= index) {
+                                vmax = index;
+                            }
+
+                            *indices = index;
+                            indices++;
+
+                            index = vertex_offset + polys[i].J;
+
+                            if (vmin >= index) {
+                                vmin = index;
+                            }
+
+                            if (vmax <= index) {
+                                vmax = index;
+                            }
+
+                            *indices = index;
+                            indices++;
+
+                            index = vertex_offset + polys[i].K;
+
+                            if (vmin >= index) {
+                                vmin = index;
+                            }
+
+                            if (vmax <= index) {
+                                vmax = index;
+                            }
+
+                            *indices = index;
+                            indices++;
+                        }
+                    }
+                }
+            }
+        }
+
+        renderer->Set_Vertex_Index_Range(vmin, vmax);
```

Comment:
seems to be (vmin, vmax - vmin + 1) and Set_Vertex_Index_Range just sets the members, doesn't do any math